### DISCUSSION TITLE
Remove "get started" allowed "working files" to expand

### DISCUSSION THREAD

**Discussion by StuartClickReturn** [brackets-cont/brackets#233] [Feature Request]
Hi all

I love this app.

i glad it was continued by the community.

BUT....

please please can some tell me how to:

- Remove "get started" section for the sidebar
- expand "working files" sidebar (the height)

or can the developers make the changes in the next update

I'm sure a lot of people use a big high def screen, not being able to increase the height of the "working files" is sooo frustrating

![Screenshot 2022-11-11 at 13 22 43](https://user-images.githubusercontent.com/117995312/201350562-9bd1705f-9d3f-4286-b669-5c18aaddc37b.png)

**Comment by charlypa** [comment]
![Brackets_7jGjMzajuj](https://user-images.githubusercontent.com/1525027/203711772-b8bc2d81-8ac1-40ce-8db8-f7a44c9c6a52.gif)

**Reply by StuartClickReturn** [reply]
i dont have this option. or rather im confused am i supposed to implement the code in the gif before i can remove "get started"?

**Reply by StuartClickReturn** [reply]
i cannot click where yours says "hello", there is no drop down menu

**Reply by StuartClickReturn** [reply]
my only says "working files"

**Reply by abose** [reply]
@StuartClickReturn UI modernization efforts are in progress as part of https://github.com/phcode-dev/phoenix
The left sidebar will be reworked after we are done with native phoenix builds in windows/mac/Linux.
